### PR TITLE
syncthing: initial addon

### PR DESCRIPTION
**Pull request by awiouy**
Hello!
Here is syncthing. At the moment, this addon uses precompiled binaries. I will update the package to compile them at a later stage.
Thank you for your attention and regards, @

### CODE REVIEW

**Comment by lrusak** [comment]
Thanks, we include golang now so it would be nice to compile this add-on from source instead of using pre compiled binaries. see the Docker add-on for compilation tips.

**Comment by chewitt** [comment]
replacement icon: http://chewitt.libreelec.tv/test/icon_syncthing.png

**Comment by awiouy** [comment]
@chewitt 
Done

@lrusak 
Syncthing is now compiled from source. Ti be consistent, I have also disabled automatic upgrades. I have tested it on LE 7.0 for RPi2, where it runs fine. This is my first go cross-compile project, and I would therefore be thankful if someone experienced could review the package.mk.

Thank you, and regards @

File: packages/addons/service/syncthing/package.mk [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+      export GOARCH=amd64
+      ;;
+    arm)
+      export GOARCH=arm
+
+      case $TARGET_CPU in
+        arm1176jzf-s)
+          export GOARM=6
+          ;;
+        cortex-a7)
```
Comment (lrusak) [inline packages/addons/service/syncthing/package.mk]: can you make this line `cortex-a7|cortex-a9)` so we can compile this for WeTek_Core and WeTek_Play.

**Pull request merged**